### ISSUE TITLE
it is too slow to start the tabby

### ISSUE THREAD

**Issue by czy2014hust** [Eugeny/tabby#6295]
Version: 1.0.176
Platform: macOS x64 21.4.0
Plugins: plugin-fig-integration
Frontend: xterm

**Comment by jenningszheng** [comment]
yes, I feel the same.

**Comment by JoshTseeee** [comment]
> yes, I feel the same. Compared with iTerm, tabby is slower, which I can tell with my eyes.

+1

**Comment by dnut** [comment]
Yeah same deal on linux. It takes several seconds to start, which is annoying for quick tasks where I'm just looking to open a new window, run a couple commands, and close it. Currently tabby is not really appropriate for this workflow because it's so slow to open and cumbersome to close.

**Comment by Margrie** [comment]
I feel the same, can someone tell why tabby starts so slow compared to the normal termininal like xterm or GNOME-Terminal. Is there an option to start tabby faster?

**Comment by dnut** [comment]
> Is there an option to start tabby faster?

Yeah! That's easy. Just use hyper, tilix, wezterm, or terminator instead of tabby. It takes 0 seconds to start if you never start it.
